src/shared/components/TextArea/TextArea.style.ts

```diff
@@ -1,67 +1,14 @@
-import { colors, sizes, transitions, typography } from '@/shared/theme'
 import styled from '@emotion/styled'
-
-export const TextAreaWrapper = styled.div`
-  width: 100%;
-`
+import resizer from '@/shared/icons/resizer.svg'
```

Comment:
This should be imported from `@/shared/icons`

Reply:
I cannot import SVG like that. Also, maybe I shouldn't put this in the `icons` directory - I don't use it as a `ReactComponent` in the first place? I'll put it in the `assets` folder. What do you think?

Reply:
ah I see, let's leave it out then